File: Server/Server.API/Models/PositionEmployeePostModel.cs
namespace Server.API.Models
{
    public class PositionEmployeePostModel
    {
        public int PositionId { get; set; }
        public int EmployeeId { get; set; }
        public DateTime EntryDateIntoOffice { get; set; }
        public bool Ismanagerial { get; set; }
    }
}

File: Client/src/app/employee/models/position-employee-post-model.ts
export class PositionEmployeePostModel {
    
        positionId: number
        entryDateIntoOffice:Date
        ismanagerial:boolean
        employeeId!:number

        
    constructor(
        positionId: number,
        entryDateIntoOffice:Date,
        ismanagerial:boolean
    ) {
        this.positionId = positionId
        this.entryDateIntoOffice = entryDateIntoOffice,
        this.ismanagerial = ismanagerial
}
}
